_pages/resources/groups/index.md

```diff
@@ -46,6 +46,8 @@ The United States is fast becoming a leader in the world-wide plain language mov
 
 - The [Quality and Productivity Commission](http://qpc.co.la.ca.us/pl.asp), established by the Board of Supervisors of the County of Los Angeles, also has its own site.
 
+- Minnesota state agencies must comply with [Governor Dayton’s Executive Order on plain language] (https://mn.gov/governor/assets/EO-14-07.pdf_tcm1055-93446.pdf). Their plain language program started in the Minnesota Department of Revenue and has spread to many other state agencies.
```

Comment:
It looks like there is an extra space after the link text which is causing the link not to display properly. Removing the space after the `]` in your link should resolve the issue.